src/hotspot/os_cpu/aix_ppc/os_aix_ppc.cpp

```diff
@@ -225,10 +226,10 @@ JVM_handle_aix_signal(int sig, siginfo_t* info, void* ucVoid, int abort_if_unrec
 
   JavaThread* thread = NULL;
   VMThread* vmthread = NULL;
-  if (os::Aix::signal_handlers_are_installed) {
+  if (PosixSignals::are_signal_handlers_installed()) {
     if (t != NULL) {
       if(t->is_Java_thread()) {
-        thread = t->as_Java_thread();
+        thread = (JavaThread*)t;
```

Comment:
Mis-merge? Please put this back to t->as_Java_thread()

Reply:
Merge issue, fixed.